README.rst

```diff
@@ -1250,6 +1246,77 @@ in the settings:
        }
    }
 
+Adding Checks and Alert settings from the device page
+-----------------------------------------------------
+
+We can add checks and define alert settings directly from the **device page**.
+
+To add a check, you just need to select an available **check type** as shown below:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/device-inline-check.gif
+  :align: center
```

Comment:
this GIF is 1 MB, can we find a way to be less heavy? Either optmize the GIF or use one or more screenshots?

Reply:
I'm unable to compress those GIFs to less than **5MB**, so I'm replacing them with screenshots.